data/json/monsters/insect_spider.json

```diff
@@ -2243,7 +2395,58 @@
         "damage_max_instance": [ { "damage_type": "stab", "amount": 15, "armor_penetration": 8 } ]
       }
     ],
-    "flags": [ "SEES", "HEARS", "SMELLS", "FLIES", "PATH_AVOID_FIRE" ]
+    "flags": [ "SEES", "HEARS", "SMELLS", "FLIES", "PATH_AVOID_FIRE" ],
+    "weakpoints": [
+      {
+        "name": "beneath the shell",
```

Comment:
I wouldn't insist on the correct `elytra`, but `beneath the outer wings` would be a bit more correct (nitpicky, I know). I'd keep the armor reduction and leave out the stun - or move that to the armored head.